### PR TITLE
LB-248: Add on_close_callback to RabbitMQ connections

### PR DESCRIPTION
**Pull request by paramsingh**
Possible fix for LB-248 - BigQuery crashing.

Added on close callbacks to the rabbitmq connections.

### CODE REVIEW

**Comment by paramsingh** [comment]
Tried to think of ways to add tests somehow for this, but couldn't really come up with anything good.

File: listenbrainz/influx-writer/influx-writer.py [hunk 1]
```diff
@@ -42,6 +42,11 @@ def __init__(self):
         self.time = 0
 
 
+    def on_connection_closed(self, connection, reply_code, reply_text):
```
Comment (mayhem) [inline listenbrainz/influx-writer/influx-writer.py]: there is no static method as a go between -- how does this work?

Comment (paramsingh) [reply listenbrainz/influx-writer/influx-writer.py]: this is how it was done in these [usage example docs](https://pika.readthedocs.io/en/0.10.0/examples/asynchronous_consumer_example.html)

but i've changed it to using a static method as done for the other callback.

Comment (mayhem) [reply listenbrainz/influx-writer/influx-writer.py]: Wow, I had no idea that was possible -- when I tried it, I had no luck. But it seems so:

https://stackoverflow.com/questions/10003833/how-does-python-distinguish-callback-function-which-is-a-member-of-a-class

(I'll remove those static functions next time I'm mucking around there)

**Pull request merged**